composer.json

```diff
@@ -27,6 +28,7 @@
         "symfony/phpunit-bridge": "^2.7 || ^3.0"
     },
     "provide": {
+        "ext-mongo": "*",
```

Comment:
From a composer update of this bundle (branch fix_php7) on php7
Is it not a "provide" limit of composer (not overloading for this property) ? 
As you can see in https://github.com/alcaeus/mongo-php-adapter/blob/master/composer.json it is ok